boefjes/Makefile

```diff
@@ -15,6 +15,7 @@ export m		# Message for alembic migration
 export revid	# Revision id to generate raw sql for
 export rev1		# Previous revision id for generating migrations
 export rev2		# New revision id for the new migration file
+export DOCKER_DEFAULT_PLATFORM=$(shell docker system info --format '{{.OSType}}/{{.Architecture}}')
```

Comment:
As far as I understand this is necessary for selecting to build linux/amd64 images on OS X? Is it always the case that docker system info return linux/amd64 on OS X or is ot also possible to run an arm64 docker?

It would also be nice to add this as a comment because it wasn't clear for me why this was set.

Reply:
This is mainly for _building_ the local images with the `images` target. When I build the image locally on my macbook, the default platform still targets `linux/amd64` which is then not usable on the same macbook unfortunately.

Reply:
Comment added 👍